src/containers/login/login.component.jsx

```diff
@@ -1,11 +1,14 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { useSelector } from "react-redux";
 import { withRouter } from "react-router-dom";
 import { googleSignInMethod } from "database";
 
 const Login = (props) => {
   const isUserLoggedIn = !!useSelector((state) => state.users.user.email);
-  if (isUserLoggedIn) props.history.push("/");
+
+  useEffect(() => {
+    if (isUserLoggedIn) props.history.push("/");
+  });
```

Comment:
This just fixes an error that we kept having in our console.log because we used routing logic inside a render function which is an anti-pattern